src/shadowbox/server/outline_shadowsocks_server.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as child_process from 'child_process';
+import * as fs from 'fs';
+import * as jsyaml from 'js-yaml';
+import * as mkdirp from 'mkdirp';
+import * as path from 'path';
+
+import * as logging from '../infrastructure/logging';
+import {AccessKey, ShadowsocksServer} from '../model/shadowsocks_server';
+
+// Runs outline-ss-server.
+export class OutlineShadowsocksServer implements ShadowsocksServer {
+  private ssProcess: child_process.ChildProcess;
```

Comment:
Since this is private you could shorten it to e.g. `process`. I personally would find this much nicer/easier to read.